src/main/java/org/togetherjava/discordbot/commands/CommandListener.java

```diff
@@ -54,6 +58,7 @@ public void onMessageReceived(@Nonnull MessageReceivedEvent event) {
       );
       executor.execute(content, context);
     } catch (CommandNotFoundException | NoSeparatorException ignored) {
+      LOG.trace("Command not foundor no separator in message {}", content);
```

Comment:
Small little typing issue.
```suggestion
      LOG.trace("Command not found or no separator in message {}", content);
```